erna/__init__.py

```diff
@@ -18,8 +18,32 @@ def build_path(row, path_to_data, extension):
     year = night[0:4]
     month = night[4:6]
     day = night[6:8]
-    return os.path.join(path_to_data, year, month, day, row.filename + extension)
-
+    res = os.path.join(path_to_data, year, month, day, row.filename + extension)
+    return res
+
+def test_drs_path(df, key):
+    numRows = len(df)
+    mask = np.ones(numRows);
+    for i in range(numRows):
+        drspath = fd[key][i]
+        if not os.path.exists(drspath):
+            mask[i] = 0
+    return df.groupby(mask)
+    
+def test_path_data(df, key):
```

Comment:
```python
def test_data_path(row, key):
    mask = df[key].apply(os.path.exists)
    df['file_exists'] = mask
    df.loc[mask, key] = df.loc[mask, key].str.replace('.fz', '.gz')
    df.loc[mask, 'file_exists'] = df.loc[mask, key].apply(os.path.exists)
```

Reply:
And missing docs

Reply:
don't you want to do df.loc[np.not(mask), key] = df.loc[np.not(mask), key].str.replace('.fz', '.gz')? I believe you want to test again with new extension for those that did not exist. Or am I getting it wrong?